scripts/sitemap.ts

```diff
@@ -122,15 +128,36 @@ async function main() {
             entry.priority = 0.8;
           }
 
+          if (story.content.component === 'landing-page') {
+            entry.changefreq = story.content.change_frequency || 'monthly';
+            entry.priority = story.content.priority || 0.7;
+          }
+
           return entry;
         })
     );
   }, []);
 
+  const chaptersArray = Object.values(chapters)
+    .flat()
+    .map((chapter) => ({
+      lastmod: chapter.lastUpdatedAt,
+      slug: chapter.slug
+    }));
+
+  const entriesHandbook = chaptersArray.map((val) => ({
+    loc: `https://significa.co/handbook/${val.slug}`,
+    lastmod: val.lastmod || new Date().toISOString(),
+    changefreq: 'monthly',
+    priority: 0.7
+  }));
+
+  const entriesWithHandbook = [...entries, ...entriesHandbook];
+
```

Comment:
what happens if the file isn't in the system? does the sitemap generation fails? or no pages appear in the sitemap? if it's the first, we should add some defensive programming for that